images/installer/Dockerfile.upi.ci

```diff
@@ -60,7 +60,7 @@ RUN python get-pip.py 'pip<21.0'
 RUN python -m pip install pyopenssl
 ENV CLOUDSDK_PYTHON=/usr/bin/python
 
-RUN python3 -m pip install yq
+RUN python3 -m pip install yq==3.1.1
```

Comment:
Maybe `<3.2.0` might be better so we can still in theory get 3.1.x bug fixes.

Reply:
Good call. I've updated the changes.